### PR TITLE
fix: build package with tsc

### PR DESCRIPTION
**Pull request by masone**
see https://github.com/carforyou/carforyou-api-client-pkg/pull/430/files

differences from the above:
- exposes a CLI script
- only needs a commonjs build as it's used uniquely as a CLI in a node environment

### CODE REVIEW

**Review by lkappeler — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -1,10 +1,13 @@
 {
   "compilerOptions": {
-    "target": "es2020",
-    "module": "esnext",
+    "module": "commonjs",
+    "target": "es5",
```
Comment (lkappeler) [inline tsconfig.json:4]: Not sure you need es5 to run it in node

Comment (masone) [reply tsconfig.json:4]: what would you do instead?

Comment (lkappeler) [reply tsconfig.json:4]: I was thinking off defining minimal node version but this one is fine as well 🙂

Comment (masone) [reply tsconfig.json:4]: it matches the recommended build we have for the server in the web projects, I'll roll with it then :)

**Pull request merged**